code/modules/mob/living/carbon/inventory.dm

```diff
@@ -140,3 +140,9 @@
 /mob/living/carbon/proc/get_holding_bodypart_of_item(obj/item/I)
 	var/index = get_held_index_of_item(I)
 	return index && hand_bodyparts[index]
+
+/mob/living/carbon/proc/get_internal_slots()

```

Comment:
Why is this a proc? This has no reason at all to be a proc when the contents are guaranteed to always be the same. Proc calls have a fairly significant overhead, far greater than the overhead of declaring a var.
Turn this into a define.
```
#define GET_INTERNAL_SLOTS(C) list(C.head, C.wear_mask)
```